src/components/Column.vue

```diff
@@ -53,10 +53,6 @@ export default {
       axios.get(`/card/column/${this.column.id}`)
         .then((response) => {
           this.cards = response.data;
-        })
-        .catch((error) => {
```

Comment:
Je kan de .catch niet zomaar weglaten?

Reply:
Tja we doen er toch niks mee. En een console.log hoor je echt niet in productie te zetten.

Reply:
true, moeten we later wel iets mee doen tho